src/js/background.js

```diff
@@ -508,14 +508,13 @@ Badger.prototype = {
 
   /**
    * Checks conditions for updating page action badge and call updateBadge
-   * @param {Object} details details object from onBeforeRequest event
+   * @param {Object} tabId The tab ID whose badge we want to update.
    */
-  updateCount: function(details) {
-    if(!this.isPrivacyBadgerEnabled(webrequest.getHostForTab(details.tabId))){
+  updateCount: function(tabId) {
```

Comment:
Currently the first half(ish) of `webrequest.js` uses `tab_id` and the second half(ish) uses `tabId`. If it's cool with you I'll just leave this and we can have a separate PR that fixes the wonky parameter and variable styling.